lib/patch-series.ts

```diff
@@ -778,7 +777,7 @@ export class PatchSeries {
         }
 
         logger.log("Publishing branch and tag");
-        if (this.project.publishToRemote && !this.options.dryRun) {
+        if (this.project.publishToRemote && !this.options.noUpdate) {
```

Comment:
I would like to keep the `--dry-run` option working, even if I have no plans on using the command-line version of GitGitGadget, ever. Therefore, I would have extended the condition, like so:

```typescript
         if (this.project.publishToRemote && !this.options.dryRun && !this.options.noUpdate) {
```

I do see, though, that you changed the command-line script to not only set `dryRun` upon the `--dry-run` option, but also `noUpdate`.

If this was done as a preparatory commit that introduces the `noUpdate` field, I would be totally on board with this, as it would make it a lot easier to review than one large patch...